src/core/ci-validation.js

```diff
@@ -17,6 +17,12 @@ class CiValidation {
                 return /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/.test(name);
             },
             message: "Please specify a valid email address. Ex. \"bot@baltimorecountymd.gov\""
+		},
+		"zipCode": {
+            test: function(name) {
```

Comment:
Is `name` the best parameter name for these?

Reply:
no fixed, if you noticed i had some bad messages in there too :weary: